mobile/components/Lists/CommunityListItem.js

```diff
@@ -0,0 +1,29 @@
+// @flow
+import * as React from 'react';
+import Avatar from '../Avatar';
+import { ListItemWithAvatar } from './ListItemWithAvatar';
+import { TextColumnContainer, Title, Subtitle } from './style';
+import type { Navigation } from '../../utils/types';
+
+type CommunityListItemType = { community: Object, navigation: Navigation };
```

Comment:
My intuition here is that it makes more sense to not require users to pass `navigation` but instead to pass `onPress`, which allows them to do whatever they hell they want on press, whether that's navigating or not?

EDIT: Just like we do in the `DirectMessageListItem`.